### PR TITLE
658 select isnt showing all elements in some cases

### PR DESCRIPTION
**Pull request by andyindahouse**
In this image you can see the bug (left image) and the fix (right image), (in first screenshot you can't see all options).

select-screenshot-test-tsx-select-1-diff

### CODE REVIEW

File: src/select.tsx [hunk 1]
```diff
@@ -106,8 +106,12 @@ const Select: React.FC<SelectProps> = ({
                 const top = selectTop + height;
                 const visibleOptions = Math.min(options.length, MAX_OPTIONS);
                 const spaceTaken = visibleOptions * 48 + PADDING_SIZE;
+                const fullOptionsSpace = options.length * 48 * PADDING_SIZE;
```
Comment (atabel) [inline src/select.tsx:109]: Is this logic right? why multiplying by PADDING_SIZE?

Comment (andyindahouse) [reply src/select.tsx:109]: I'll change it and test it again

Comment (andyindahouse) [reply src/select.tsx]: Done

File: src/__stories__/select-story.tsx [hunk 2]
```diff
@@ -7,14 +7,21 @@ export default {
...
 };
 
-const fruitOptions = fruitEntries.map(([text, value]) => ({text, value}));
+const fruitOptions = [
+    ...fruitEntries,
+    ...fruitEntries.map(([text, value]) => [text + 1, value + 1]),
+    ...fruitEntries.map(([text, value]) => [text + 2, value + 2]),
+    ...fruitEntries.map(([text, value]) => [text + 3, value + 3]),
+    ...fruitEntries.map(([text, value]) => [text + 4, value + 4]),
+    ...fruitEntries.map(([text, value]) => [text + 5, value + 5]),
```
Comment (atabel) [inline src/__stories__/select-story.tsx:16]: I'd just add more fruits to the array. here you have some (generated by Copilot):
```js
    ['Pineapple', 'pineapple'],
    ['Strawberry', 'strawberry'],
    ['Watermelon', 'watermelon'],
    ['Grapes', 'grapes'],
    ['Mango', 'mango'],
    ['Peach', 'peach'],
    ['Cherry', 'cherry'],
    ['Raspberry', 'raspberry'],
    ['Blueberry', 'blueberry'],
    ['Kiwi', 'kiwi'],
    ['Pomegranate', 'pomegranate'],
    ['Papaya', 'papaya'],
    ['Coconut', 'coconut'],
    ['Avocado', 'avocado'],
    ['Cantaloupe', 'cantaloupe'],
    ['Mandarin', 'mandarin'],
    ['Melon', 'melon'],
    ['Tangerine', 'tangerine'],
    ['Tomato', 'tomato'],
```

**Review by atabel — APPROVED** [review]

**Pull request merged**